src/modules/Dropdown/Dropdown.js

```diff
@@ -751,7 +751,7 @@ export default class Dropdown extends Component {
   renderText = () => {
     const { multiple, placeholder, search, text } = this.props
     const { searchQuery, value, open } = this.state
-    const hasValue = multiple ? !_.isEmpty(value) : !!value
+    const hasValue = multiple ? !_.isEmpty(value) : value !== ''
```

Comment:
When `value` is `null` or `undefined` then `hasValue` will be true.  We should add a nil check in here to account for those:

```jsx
const hasValue = multiple 
  ? !_.isEmpty(value) 
  : !_.isNil(value) || value !== ''